app/views/installations/index.html.haml

```diff
@@ -1,20 +1,395 @@
 #installation.row
   .col-md-12.pagecontainer
-    .pagetitle
-      %h3 Installation and configuration
-    .panel.panel-default.codeview
-      .panel-heading Client Configuration
-      .panel-body
-        %p To start sending events, configure your client with the following DSN value:
-        .codepanel
-          %span.line https://451cf228845b497b919281b158209a59:157cbe7bf9d34b67b906dea4d3e849d0@app.feedbacker.com/12798
-        %br/
-        %p If this is a client that only needs public access (like raven-js), you can use the following public DSN:
-        .codepanel
-          %span.line https://3291842f716643f2968b11e3c86113f9@app.feedbacker.com/12798
-    .panel.panel-default.tileview
-      .panel-heading Platforms
-      .panel-body
-        %a.tile.text-center{:href => "installations/show"}
-          %img{:src => image_path("icon_javascript.png")}/
-          %span Javascript
\ No newline at end of file
+    %br
+    .bs-example{"data-example-id" => "simple-nav-justified"}
```

Comment:
e nevoie de clasa asta bs-example aici? Asta nu are nici o valoare decat in bootstrap example documentation. Care e rostul ei? Daca ai pus-o doar asa din copy paste incearca sa o schimbi chiar daca nu ai class pe ea.